QRCoder/QRCodeData.cs

```diff
@@ -152,7 +152,11 @@ public byte[] GetRawData(Compression compressMode)
         try
         {
             //Add header - signature ("QRR")
+#if NETCOREAPP2_1_OR_GREATER || NETSTANDARD2_1
+            targetStream.Write([0x51, 0x52, 0x52, 0x00]);
```

Comment:
Just as note: for .NET less than 8 this will still allocate the array, as the collection builder, etc. isn't available in .NET, so Roslyn can't use these types and produces "fallback-code". 
The change is still good, as it clearer to read. Thanks.